src/components/select/SelectConnected.tsx

```diff
@@ -95,7 +96,7 @@ export class SelectConnected extends React.Component<ISelectProps & ISelectSpeci
 
     componentWillUpdate(nextProps: ISelectProps): any {

```

Comment:
won't `componentWillUpdate` be deprecated? It's a good opportunity here to use something else

Reply:
`componentDidUpdate ` can do the job